src/styles/blocks/icon.scss

```
@@ -0,0 +1,53 @@
+.icon {
+  display: block;
+  width: $icon-width;
+  padding-bottom: $icon-width;
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+  text-decoration: none;
+
+  &--menu {
+    background-image: url(../images/menu.svg);
+  }
+

```

```suggestion
  &--menu {
    background-image: url(../images/menu.svg);
    
    &:hover {
      background-image: url(../images/menu_hover.svg);
    }
  }
```
to make the menu icon change on hover